framework/Source/Sparkles.swift

```diff
@@ -28,6 +28,18 @@ public class Sparkles: OperationGroup {
         didSet { perlinNoiseEffect.time = time }
     }
     // LightExtractor
+    public var faceMaskImage: UIImage? {
+        didSet {
+            guard faceMaskImage != oldValue else { return }
+            let maskImage: UIImage = faceMaskImage ?? .emptyMaskImage
```

Comment:
Hi @yyjim, as per our previous discussion in another PR: https://github.com/cardinalblue/GPUImage3/pull/23#discussion_r1645232634

I've updated the implementation to handle the nil faceMaskImage case.
Please help to review the PR again when you're available, thanks!